infra/gravity/testcontext.go

```diff
@@ -217,7 +218,12 @@ func (msg progressMessage) Save() (row map[string]bigquery.Value, insertID strin
 func (c *TestContext) updateStatus(status string) {
 	c.status = status
 
-	log := c.Logger().WithFields(logrus.Fields{"param": xlog.ToJSON(c.param), "name": c.name})
+	log := c.Logger().WithFields(logrus.Fields{
+		"param":   xlog.ToJSON(c.param),
+		"name":    c.name,
+		"version": robotest.Version,
+		"commit":  robotest.GitCommit,
```

Comment:
I'm not entirely sure that I like adding these to the top level status logs (RUNNING, PASSED, FAILED).  However I do want version information close to the param information, as the combination of the two should be most everything needed to reproduce issues.